src/main/java/net/contargo/types/telephony/validation/ValidPhoneNumberValidator.java

```diff
@@ -49,6 +50,11 @@ public boolean isValid(String value, ConstraintValidatorContext cvc) {
                 reportConstraintViolation(cvc, "{PHONE_NUMBER_TOO_LARGE}");
                 isValid = false;
             }
+
+            if (phoneNumber.isValidNumber()) {
```

Comment:
Shouldn't this be negated? This block now is entered for a valid number and than isValid is set false. Does not seem right to me.